mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,274 @@
+    <string name="add_tab">Add Tab</string>
+    <!-- Content description (not visible, for screen readers etc.): Close tab button. Closes the current session when pressed -->
+    <string name="close_tab">Close tab</string>
+    <!-- Content description (not visible, for screen readers etc.): Opens the tab menu when pressed -->
+    <string name="tab_menu">Tab menu</string>
+    <!-- Content description (not visible, for screen readers etc.): Close tab button. Favicon for the session's site -->
+    <string name="favicon_content_description">Bookmark</string>
+    <!-- Button in the current session menu. Deletes the session when pressed -->
+    <string name="current_session_delete">Delete</string>
+    <!-- Button in the current session menu. Saves the session when pressed -->
+    <string name="current_session_save">Save</string>
+    <!-- Button in the current session menu. Opens the share menu when pressed -->
+    <string name="current_session_share">Share</string>
+    <!-- Content description (not visible, for screen readers etc.): Title icon for current session menu -->
+    <string name="current_session_image">Current session image</string>
+    <!-- Text for the button to save a session -->
+    <string name="session_save">Save session</string>
+    <!-- Text for the button to delete a session -->
+    <string name="session_delete">Delete session</string>
+    <!-- Text for the button to delete a single session -->
+    <string name="session_item_delete">Delete</string>
+    <!-- Text to tell the user how many more tabs this session has.
+        The first parameter is how many extra tabs the session has. -->
+    <string name="session_items_more">%1$d sites…</string>
+
+    <!-- History -->
+    <!-- Text for the button to clear all history -->
+    <string name="history_delete_all">Delete history</string>
+    <!-- Text for the button to delete a single history item -->
+    <string name="history_delete_item">Delete</string>
+    <!-- Text for the button to clear selected history items. The first parameter
+        is the number of items you have selected -->
+    <string name="history_delete_some">Delete %1$d items</string>
+    <!-- Text for the header that groups the history for Today -->
+    <string name="history_today">Today</string>
+    <!-- Text for the header that groups the history the past week -->
+    <string name="history_this_week">This Week</string>
+    <!-- Text for the header that groups the history the past month -->
+    <string name="history_this_month">This Month</string>
+    <!-- Text for the header that groups the history older than the last month -->
+    <string name="history_older">Older</string>
+    <!-- Text displayed in a notification when the user enters full screen mode -->
+    <string name="full_screen_notification">Entering full screen mode</string>
+
+    <!-- Crashes -->
+    <!-- Title text displayed on the tab crash page -->
```

Comment:
Please explain what placeholder will be replaced with

Reply:
Think this was delivered in another issue from @sblatz